change/@ni-nimble-components-2c319a68-618f-4ea0-89c6-e3ef1bfbccd6.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
not a very scientific test but was curious if all the extra notifiers could effect perf, but looks like on scrolling it might be a little better, maybe because we don't need to update all cellstate anymore

before (10-12 ms of work per frame)

![image](https://github.com/ni/nimble/assets/1588923/bac1f5d6-4696-4553-900b-6119febb6585)


after (7-9 ms of work per frame)

![image](https://github.com/ni/nimble/assets/1588923/8c96faa7-e436-49f8-a875-f18f72a3f11f)